sensor/upgrader/preflight/namespace.go

```diff
@@ -34,14 +37,16 @@ func namespaceAllowed(resource *k8sobjects.ObjectRef) bool {
 	if matchesException(resource) {
 		return true
 	}
-	return resource.Namespace == "" || resource.Namespace == common.Namespace
+	logging.Infof("Object \"%v\" in namespace \"%s\"\n in pod with namespace \"%s\", function will return \"%s\"", resource, resource.Namespace, pods.GetPodNamespace(pods.NoSATokenNamespace), strconv.FormatBool((resource.Namespace == "") || (resource.Namespace == pods.GetPodNamespace(pods.NoSATokenNamespace))))
```

Comment:
Do you want to keep this log line or it's for tracing? Also, there's `%q` which reads simpler as `\"%s\"`.

Reply:
I'm still doing some manual tests to be thorough after which I'll remove this and merge the PR.